app.js

```diff
@@ -29,20 +29,20 @@ var csp = require('./lib/csp')
 // server setup
 var app = express()
 var server = null
-if (config.useSSL) {
+if (config.get('useSSL')) {
```

Comment:
I'll not comment this on each item, but I think we should also "namespace" this a little more. And perhaps while we're at it we could use the more correct "useTLS" because luckily nobody is using SSL anymore.

Reply:
This would break everyone's setup that uses this variable. I mean, we can do that' but I wanted to mention it before we do that.